### PR TITLE
Add FIPS support for generic RPM package

### LINKED ISSUE #367: Add RHEL 8 FIPS mode compatibility
Hello team,

We have found that due to this:
https://bugzilla.redhat.com/show_bug.cgi?id=1728031

The current packages cant be installed in RHEL 8 if FIPS mode is enabled, so we need to modify our package building method to fix this problem.

Regards, 
Daniel Folch

### PR DESCRIPTION
**Pull request by BraulioV**
Hi team,

This issue closes #367. All the details can be found in the issue's comments.

Regards.

### CODE REVIEW

File: rpms/CentOS/6/i386/Dockerfile [hunk 1]
```diff
@@ -7,10 +7,37 @@ RUN yum -y install util-linux-ng centos-release-scl \
     gcc-multilib make git openssh-clients rpm-build \
     sudo gnupg automake autoconf libtool \
     policycoreutils-python yum-utils epel-release \
-    redhat-rpm-config rpm-devel
+    redhat-rpm-config rpm-devel autopoint gettext \
+    zlib zlib-devel nspr nspr-devel \
+    nss nss-devel kenel-headers magic magic-devel \
+    db4 db4-devel zlib zlib-devel rpm-build bison \
```
Comment (okynos) [inline rpms/CentOS/6/i386/Dockerfile]: Zlib and zlib-devel are twice, rpm-build too.

File: rpms/CentOS/6/x86_64/Dockerfile [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
 RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6
-RUN yum -y install centos-release-scl gcc make git \
+RUN yum install -y centos-release-scl gcc make git \
     openssh-clients rpm-build sudo gnupg \
     automake autoconf libtool policycoreutils-python \
-    yum-utils epel-release redhat-rpm-config rpm-devel
+    yum-utils epel-release redhat-rpm-config rpm-devel \
+    autopoint gettext zlib zlib-devel nspr nspr-devel \
+    nss nss-devel kenel-headers magic magic-devel \
+    db4 db4-devel zlib zlib-devel rpm-build bison \
```
Comment (okynos) [inline rpms/CentOS/6/x86_64/Dockerfile]: Same here.

Comment (BraulioV) [reply rpms/CentOS/6/x86_64/Dockerfile]: Fixed in https://github.com/wazuh/wazuh-packages/pull/378/commits/90204700decee13e474b57e790795ab160dc1a52

File: rpms/CentOS/6/i386/Dockerfile (hunk 1 above)
Comment (BraulioV) [reply rpms/CentOS/6/i386/Dockerfile]: Fixed in https://github.com/wazuh/wazuh-packages/pull/378/commits/90204700decee13e474b57e790795ab160dc1a52

**Pull request merged**